continuous_integration/recipes/dask/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev') %}
+{% set nightly_version = version + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version=environ.get('CONDA_PY', 36) %}
+
+
+package:
+  name: dask
```

Comment:
Do we want this here given it depends `distributed` nightly packages will be needed or should we save this for the distributed repo when those nightlies are setup?

Reply:
Ah was making sure that was what you meant in https://github.com/dask/dask/pull/8469#discussion_r766209711 - yeah we can remove this and handle it in the distributed nightlies 😄

Reply:
Ah sorry didn't see that. Responded above as well